project-base/app/config/graphql/types/ModelType/Slider/SliderItem.types.yaml

```diff
@@ -23,6 +23,15 @@ SliderItem:
             gtmCreative:
                 type: "String"
                 description: "GTM creative"
+            description:
+                type: "String!"
+                description: "Slider description"
+            rgbBackgroundColor:
+                type: "String"
+                description: "RGB color code for the background of the slider description box"
+            opacity:
+                type: "Float"
+                description: "Opacity level for the background color of the slider description box"
```

Comment:
all these changes should be actually in the `frontend-api` package. This opens the door to more code movements :smile: If you feel like it, could you please move _everything_ related to the slider items to the proper packages? I see all the API related code (types and query) is in the project base, also there are some entity extensions, `SliderItemFormTypeExtension`, etc. If you do not feel like it, please move at least the API type definitions :pray: 

We can discuss this if you need :wink: